### PR TITLE
Fix README.md for autosuggestions and history-substring-search

### LINKED ISSUE #1441: Ambiguous pronouns in module READMEs
### Description

In module READMEs, some pronouns may be ambiguous. For example, in the [autosuggestions module](https://github.com/sorin-ionescu/prezto/tree/master/modules/autosuggestions)'s README.md:

> If this module is used in conjuncture with the syntax-highlighting module, it must be loaded after it.

Some people may not be able to tell what the first 'it' refers to.
For example, does the sentence mean:

> If this module is used in conjuncture with the syntax-highlighting module, this module must be loaded after it.

Or is the sentence supposed to mean:

> If this module is used in conjuncture with the syntax-highlighting module, it must be loaded after this module.

Fixing this issue will minimize confusion and prevent configuration mistakes.

### PR DESCRIPTION
**Pull request by sergiorussia**
Fixes #1441 and #1444 

## Proposed Changes

  - Fixed documentation regression for autosuggestions module
  - Fixed the incorrect use of 'conjuncture' by replacing it with 'conjunction'.

### CODE REVIEW

**Comment by sergiorussia** [comment]
Regarding changes in autosuggestions module refer to my comments in #1444.
TL;DR: "it should be loaded after it" meant "this module should be loaded after XYZ module"

**Comment by sergiorussia** [comment]
@belak could you take a look?

File: modules/autosuggestions/README.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ of a previously entered command and Zsh suggests commands as you type based on
 history and completions.
 
 If this module is used in conjunction with the *syntax-highlighting* module,
-the *syntax-highlighting* module must be loaded **after** this module.
+this module must be loaded **after** the *syntax-highlighting* module.
 
 If this module is used in conjunction with the *history-substring-search*
-module, the *history-substring-search* module must be loaded **after** this
+module, this module must be loaded **after** the the *history-substring-search*
```
Comment (belak) [inline modules/autosuggestions/README.md]: Double `the` should be removed

File: modules/history-substring-search/README.md [hunk 2]
```diff
@@ -6,8 +6,8 @@ the [Fish shell][2]'s history search feature, where the user can type in any
 part of a previously entered command and press up and down to cycle through
 matching commands.
 
-If this module is used in conjuncture with the *syntax-highlighting* module, it
-must be loaded **after** it.
+If this module is used in conjunction with the *syntax-highlighting* module,
+this module must be loaded **after** the the *syntax-highlighting* module.
```
Comment (belak) [inline modules/history-substring-search/README.md]: Same double `the` should be cleaned up here

**Comment by sergiorussia** [comment]
@belak , done

**Comment by belak** [comment]
Looks good, thanks!
